issue-cert.sh

```diff
@@ -14,7 +15,7 @@ extendedKeyUsage = serverAuth, clientAuth
 subjectAltName = DNS:${HOST}
 "
 cat <<< "$local_openssl_config" > node.ext
-openssl x509 -req -in $HOST.csr -CA ca/ca.pem -CAkey ca/ca.key -CAcreateserial -out $HOST.pem -days 365 -sha256 \
+openssl x509 -req -in $HOST.csr -CA $NETWORK/ca.pem -CAkey $NETWORK/ca.key -CAcreateserial -out $HOST.pem -days 365 -sha256 \
```

Comment:
`-CAcreateserial` can be replaced with `-CAserial $NETWORK/ca.srl` (or removed completely since it should default to this file). 
I'm not sure what the benefit of using the serial is since everyone will be assigned the same serial number. Maybe  auto generate the */ca.srl files as well if uniqueness is desired here?

Reply:
Shouldn't have been checked in I guess